client/src/shared/components/Table/TableOld.tsx

```diff
@@ -0,0 +1,73 @@
+/* eslint-disable @typescript-eslint/strict-boolean-expressions */
+/* eslint-disable multiline-ternary */
+import React, { useState } from 'react';
+import type { ReactNode } from 'react';
+import HeaderTitle from './HeaderTitle';
+
+export enum TableSortEnum {
+  ASC,
+  DESC,
+}
+export interface TableHeader {
+  text: string;
+  onClick?: () => void;
+}
+
+export interface TableProps {
+  header: TableHeader[];
+  checkbox?: boolean;
+  children: ReactNode;
+  sortable?: boolean;
+}
+
+const TableOld: React.FC<TableProps> = ({
+  header,
+  children,
+  checkbox = true,
+  sortable = true,
+}: TableProps) => {
+  const [sort, setSort] = useState({
+    index: -1,
+    sortBy: TableSortEnum.ASC,
+  });
+
+  const handleSortChange = (uid: number): void => {
+    setSort((curr) => ({
+      index: uid,
+      sortBy:
+        curr.index === uid
+          ? curr.sortBy === TableSortEnum.ASC
+            ? TableSortEnum.DESC
+            : TableSortEnum.ASC
+          : TableSortEnum.ASC,
+    }));
+  };
+
+  return (
+    <div className="overflow-auto text-neutral-900 flex justify-center rounded-[5px]">
+      <table className="text-left w-full">
+        <thead className="bg-neutral-200">
+          <tr>
+            <th className="p-4">
+              {checkbox && <input type="checkbox" className="h-5 w-5 hover:bg-sky-700" />}
+            </th>
```

Comment:
Instead of creating another table component, I think you can modify the old one. 
You can add a prop here to make the header dynamic (would rows have a column for checkbox or not):

```suggestion
            {selectableRows && <th className="p-4">
              {checkbox && <input type="checkbox" className="h-5 w-5 hover:bg-sky-700" />}
            </th>
            }
```

Just set a default to that prop in this file and adjust it accordingly on the component you're using it on.